### ISSUE TITLE
container crashing

### ISSUE THREAD

**Issue by initze** [initze/thaw-slump-segmentation#31]
setup_raw_data crash on setup_raw_data.py

```
    return [func(*args, **kwargs)
  File "setup_raw_data.py", line 67, in preprocess_directory
    success_state['label'] = vector_to_raster_mask(image_dir)
  File "/isipd/projects/p_aicore_pf/initze/code/test/lib/data_pre_processing/utils.py", line 133, in vector_to_raster_mask
    os.remove(maskfile)
FileNotFoundError: [Errno 2] No such file or directory: '/isipd/projects/p_aicore_pf/initze/processing_container/input/20200814_044131_1026/mask.tif'
```

**Comment by initze** [comment]
Bug comes from clunky usage of gdal calls

anaconda: $CONDA_PREFIX/python
linux pure: /bin/sh/python3

**Comment by initze** [comment]
```
Singularity> python setup_raw_data.py --data_dir $PROC_DIR --log_dir $PROC_DIR/logs --gdal_bin /usr/bin --gdal_path /usr/bin
[2022-03-23T21:35:02.653+01:00] uc2.setup_raw_data INFO: ###########################
[2022-03-23T21:35:02.653+01:00] uc2.setup_raw_data INFO: # Starting Raw Data Setup #
[2022-03-23T21:35:02.653+01:00] uc2.setup_raw_data INFO: ###########################
[2022-03-23T21:35:09.175+01:00] uc2.setup_raw_data.20200814_044131_1026 INFO: Starting preprocessing 20200814_044131_1026
[2022-03-23T21:35:09.177+01:00] uc2.setup_raw_data.20200814_044131_1026.utils INFO: No "_clip" naming found. Assume renaming not necessary
[2022-03-23T21:35:09.476+01:00] uc2.setup_raw_data.20200814_044131_1026.utils INFO: Called `python /usr/bin/gdal_merge.py -createonly -init 0 -o /isipd/projects/p_aicore_pf/initze/processing_container/input/20200814_044131_1026/mask.tif -ot Byte -co COMPRESS=DEFLATE /isipd/projects/p_aicore_pf/initze/processing_container/input/20200814_044131_1026/20200814_044131_1026_3B_AnalyticMS_SR.tif`
[2022-03-23T21:35:09.476+01:00] uc2.setup_raw_data.20200814_044131_1026.utils ERROR: > /bin/sh: 1: python: not found
[2022-03-23T21:35:10.302+01:00] uc2.setup_raw_data.20200814_044131_1026.utils INFO: Called `/usr/bin/gdal_translate -of GTiff -ot Byte -co COMPRESS=DEFLATE -b 1 /isipd/projects/p_aicore_pf/initze/processing_container/input/20200814_044131_1026/mask.tif /isipd/projects/p_aicore_pf/initze/processing_container/input/20200814_044131_1026/20200814_044131_1026_mask.tif`
[2022-03-23T21:35:10.303+01:00] uc2.setup_raw_data.20200814_044131_1026.utils ERROR: > ERROR 4: /isipd/projects/p_aicore_pf/initze/processing_container/input/20200814_044131_1026/mask.tif: No such file or directory
[2022-03-23T21:35:10.371+01:00] uc2.setup_raw_data.20200814_044131_1026.utils INFO: Called `/usr/bin/gdal_rasterize -l 20200814_044131_1026 -a label /isipd/projects/p_aicore_pf/initze/processing_container/input/20200814_044131_1026/20200814_044131_1026.shp /isipd/projects/p_aicore_pf/initze/processing_container/input/20200814_044131_1026/20200814_044131_1026_mask.tif`
[2022-03-23T21:35:10.371+01:00] uc2.setup_raw_data.20200814_044131_1026.utils ERROR: > ERROR 1: Size and resolutions are missing
```

**Comment by initze** [comment]
Change python calls to "python3" - both pure and anaconda should be working

**Comment by initze** [comment]
Pyproj

```
File "/usr/local/lib/python3.8/dist-packages/joblib/externals/loky/process_executor.py", line 288, in __call__
    return self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.8/dist-packages/joblib/_parallel_backends.py", line 595, in __call__
    return self.func(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/joblib/parallel.py", line 262, in __call__
    return [func(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/joblib/parallel.py", line 262, in <listcomp>
    return [func(*args, **kwargs)
  File "setup_raw_data.py", line 74, in preprocess_directory
    success_state['tcvis'] = get_tcvis_from_gee(image_dir,
  File "/thaw-slump-segmentation/lib/data_pre_processing/earthengine.py", line 59, in get_tcvis_from_gee
    geom_4326, coords, epsg = ee_geom_from_image_bounds(impath, buffer=buffer)
  File "/thaw-slump-segmentation/lib/data_pre_processing/earthengine.py", line 40, in ee_geom_from_image_bounds
    transformer = Transformer.from_crs(epsg, "epsg:4326")
  File "/usr/local/lib/python3.8/dist-packages/pyproj/transformer.py", line 567, in from_crs
    cstrencode(CRS.from_user_input(crs_from).srs),
  File "/usr/local/lib/python3.8/dist-packages/pyproj/crs/crs.py", line 479, in from_user_input
    return cls(value, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/pyproj/crs/crs.py", line 326, in __init__
    self._local.crs = _CRS(self.srs)
  File "pyproj/_crs.pyx", line 2352, in pyproj._crs._CRS.__init__
pyproj.exceptions.CRSError: Invalid projection: EPSG:None: (Internal Proj Error: proj_create: crs not found)
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "setup_raw_data.py", line 120, in <module>
    Parallel(n_jobs=args.n_jobs)(delayed(preprocess_directory)(image_dir, args, log_path, args.nolabel) for image_dir in dir_list)
  File "/usr/local/lib/python3.8/dist-packages/joblib/parallel.py", line 1056, in __call__
    self.retrieve()
  File "/usr/local/lib/python3.8/dist-packages/joblib/parallel.py", line 935, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/usr/local/lib/python3.8/dist-packages/joblib/_parallel_backends.py", line 542, in wrap_future_result
    return future.result(timeout=timeout)
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 444, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
pyproj.exceptions.CRSError: Invalid projection: EPSG:None: (Internal Proj Error: proj_create: crs not found)
```

**Comment by initze** [comment]
fixed with #33
